projects/storefrontapp/src/app/app.module.ts

```diff
@@ -28,16 +28,24 @@ if (!environment.production) {
   devImports.push(StoreDevtoolsModule.instrument());
 }
 
+let additionalImports = [];
```

Comment:
I kind of liked having these inline in the `imports` array :)

Reply:
I couldn't get rid of the `position 4` errors :(
I will try again to improve that.